docs/v0.6.0/webhooks.md

```diff
@@ -18,3 +18,14 @@ rio run -p 80 --build-webhook-secret=githubtoken https://github.com/example/exam
 
 3. Go to your Github repo, it should have webhook configured to point to one of our webhook service.
 
+# Webhook for Riofile
+
+Setup webhook and private git clone secret for git repository that contains Riofile:
+
+```bash
+$ rio up --build-clone-secret gitsecret --build-webhook-secret webhook https://github.com/example/example
+```
+
+For how to add git and webhook secret, check [here](./continuous-deployment.md).
```

Comment:
Should this be `webhooks.md` ?

Reply:
The docs for adding secrets actually lives in continuous-deployment.md. I think we can keep it there.